[2025-07-13T10:55:04.125+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-13T10:55:04.142+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: employee_ingestion_dag.split_and_save_data scheduled__2025-07-13T10:54:00+00:00 [queued]>
[2025-07-13T10:55:04.150+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: employee_ingestion_dag.split_and_save_data scheduled__2025-07-13T10:54:00+00:00 [queued]>
[2025-07-13T10:55:04.150+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-13T10:55:04.160+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): split_and_save_data> on 2025-07-13 10:54:00+00:00
[2025-07-13T10:55:04.168+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=96) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-13T10:55:04.169+0000] {standard_task_runner.py:72} INFO - Started process 100 to run task
[2025-07-13T10:55:04.170+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'employee_ingestion_dag', 'split_and_save_data', 'scheduled__2025-07-13T10:54:00+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_pipeline.py', '--cfg-path', '/tmp/tmpflh7zy32']
[2025-07-13T10:55:04.171+0000] {standard_task_runner.py:105} INFO - Job 121: Subtask split_and_save_data
[2025-07-13T10:55:04.223+0000] {task_command.py:467} INFO - Running <TaskInstance: employee_ingestion_dag.split_and_save_data scheduled__2025-07-13T10:54:00+00:00 [running]> on host 1f767f70b1bf
[2025-07-13T10:55:04.304+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='employee_ingestion_dag' AIRFLOW_CTX_TASK_ID='split_and_save_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-13T10:54:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-13T10:54:00+00:00'
[2025-07-13T10:55:04.305+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-13T10:55:04.745+0000] {logging_mixin.py:190} INFO - Processing file: employee_data2_with_errors_chunk_14.csv
[2025-07-13T10:55:04.746+0000] {logging_mixin.py:190} INFO - Total rows: 2413, Invalid rows: 0
[2025-07-13T10:55:04.761+0000] {logging_mixin.py:190} INFO - File moved to good_data: /opt/airflow/data/good_data/employee_data2_with_errors_chunk_14.csv
[2025-07-13T10:55:04.763+0000] {logging_mixin.py:190} INFO - Original file removed: /opt/airflow/data/raw_data/employee_data2_with_errors_chunk_14.csv
[2025-07-13T10:55:04.763+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-13T10:55:04.773+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-13T10:55:04.774+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=employee_ingestion_dag, task_id=split_and_save_data, run_id=scheduled__2025-07-13T10:54:00+00:00, execution_date=20250713T105400, start_date=20250713T105504, end_date=20250713T105504
[2025-07-13T10:55:04.826+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-13T10:55:04.841+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-13T10:55:04.843+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
